config/jobs/kubernetes-sigs/cluster-api-provider-docker/cluster-api-provider-docker-presubmits.yaml

```diff
@@ -8,6 +8,10 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190703-1f4d616-master
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190703-1f4d616-experimental
```

Comment:
Stolen from https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml#L42-L44

Reply:
FYI this will track what versions of tools test-infra has, versus what version of tools kubernetes/kubernetes has on the master branch

that may be what you want, but just a heads up there. these images are regularly updated.

Reply:
This is very likely _not_ what you want, and may lead to us breaking you at any time. Why do you want `experimental`?

Reply:
This was copied from another job. Do you have an alternate tag that you would like me to use?

Reply:
The reason for the image name change is to be able to use docker in docker. So we need docker binaries built into the image.

Reply:
`master` should have docker binaries in it already:

```
ktbry@ktbry2 ~> docker run --rm -it --entrypoint /bin/bash gcr.io/k8s-testimages/kubekins-e2e:v20190703-1f4d616-master
root@599eae9b66f7:/workspace# docker version
Client:
 Version:           18.09.6
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        481bc77
 Built:             Sat May  4 02:36:00 2019
 OS/Arch:           linux/amd64
 Experimental:      false
```

Reply:
I might have done something wrong. Thanks for checking this for me :)
I've reverted the image name change. PTAL